renderer/hooks/editor/use-conversion-id.tsx

```diff
@@ -23,6 +23,8 @@ export const ConversionIdContextProvider = (props: PropsWithChildren<Record<stri
     setConversionId(id || savedConversionId);
   };
 
+  // TODO: Fix this.
+  // eslint-disable-next-line react/jsx-no-constructed-context-values
```

Comment:
This actually might be the reason for some random re-renders I was seeing. I'll push a commit to fix it with a useCallback